### ISSUE TITLE
"Unused Argument" when (re) installing Keras

### ISSUE THREAD

**Issue by Austin-Fournier** [rstudio/keras#977]
I uninstalled and reinstalled the Keras package for R (as well as my Anaconda installation) to start using it with my gpu. However, when I run the following:
```
install_keras(method = 'conda', tensorflow='gpu')
```
I get an error:
```
Error in conda_install(envname, packages, conda = conda, ...) : unused argument (python_version = conda_python_version)
```
I'm aware of [one previous user with this problem](https://github.com/rstudio/keras/issues/869), but I tried doing what worked for them and it didn't work for me. 

My session info is as follows:
```
> devtools::session_info()
- Session info ------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.3 (2019-03-11)
 os       Windows >= 8 x64            
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United States.1252  
 ctype    English_United States.1252  
 tz       America/Los_Angeles         
 date     2020-02-03                  

- Packages ----------------------------------------------------------------------------------------------------
 package     * version date       lib source        
 assertthat    0.2.0   2017-04-11 [1] CRAN (R 3.5.2)
 backports     1.1.3   2018-12-14 [1] CRAN (R 3.5.2)
 base64enc     0.1-3   2015-07-28 [1] CRAN (R 3.5.2)
 callr         3.2.0   2019-03-15 [1] CRAN (R 3.5.3)
 cli           1.0.1   2018-09-25 [1] CRAN (R 3.5.2)
 crayon        1.3.4   2017-09-16 [1] CRAN (R 3.5.2)
 desc          1.2.0   2018-05-01 [1] CRAN (R 3.5.2)
 devtools      2.0.1   2018-10-26 [1] CRAN (R 3.5.2)
 digest        0.6.18  2018-10-10 [1] CRAN (R 3.5.2)
 fs            1.2.6   2018-08-23 [1] CRAN (R 3.5.2)
 generics      0.0.2   2018-11-29 [1] CRAN (R 3.5.2)
 glue          1.3.1   2019-03-12 [1] CRAN (R 3.5.3)
 jsonlite      1.6     2018-12-07 [1] CRAN (R 3.5.2)
 keras       * 2.2.5.0 2019-10-08 [1] CRAN (R 3.5.3)
 knitr         1.22    2019-03-08 [1] CRAN (R 3.5.3)
 lattice       0.20-38 2018-11-04 [2] CRAN (R 3.5.3)
 magrittr      1.5     2014-11-22 [1] CRAN (R 3.5.2)
 Matrix        1.2-15  2018-11-01 [2] CRAN (R 3.5.3)
 memoise       1.1.0   2017-04-21 [1] CRAN (R 3.5.2)
 pkgbuild      1.0.2   2018-10-16 [1] CRAN (R 3.5.2)
 pkgload       1.0.2   2018-10-29 [1] CRAN (R 3.5.2)
 prettyunits   1.0.2   2015-07-13 [1] CRAN (R 3.5.2)
 processx      3.3.0   2019-03-10 [1] CRAN (R 3.5.3)
 ps            1.3.0   2018-12-21 [1] CRAN (R 3.5.2)
 R6            2.4.0   2019-02-14 [1] CRAN (R 3.5.3)
 Rcpp          1.0.0   2018-11-07 [1] CRAN (R 3.5.2)
 remotes       2.0.2   2018-10-30 [1] CRAN (R 3.5.2)
 reticulate    1.11.1  2019-03-06 [1] CRAN (R 3.5.3)
 rlang         0.3.1   2019-01-08 [1] CRAN (R 3.5.2)
 rprojroot     1.3-2   2018-01-03 [1] CRAN (R 3.5.2)
 rstudioapi    0.9.0   2019-01-09 [1] CRAN (R 3.5.2)
 sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.5.2)
 tensorflow    2.0.0   2019-10-02 [1] CRAN (R 3.5.3)
 testthat      2.0.1   2018-10-13 [1] CRAN (R 3.5.2)
 tfruns        1.4     2018-08-25 [1] CRAN (R 3.5.3)
 usethis       1.4.0   2018-08-14 [1] CRAN (R 3.5.2)
 whisker       0.3-2   2013-04-28 [1] CRAN (R 3.5.2)
 withr         2.1.2   2018-03-15 [1] CRAN (R 3.5.2)
 xfun          0.5     2019-02-20 [1] CRAN (R 3.5.3)
 yaml          2.2.0   2018-07-25 [1] CRAN (R 3.5.2)
 zeallot       0.1.0   2018-01-28 [1] CRAN (R 3.5.3)

[1] C:/Users/Austi/Documents/R/win-library/3.5
[2] C:/Program Files/R/R-3.5.3/library
```

**Comment by dfalbel** [comment]
Can you try updating your `reticulate` version?

**Comment by Austin-Fournier** [comment]
My apologies, I fixed it earlier then forgot to come back and update my post. .

Updating ```reticulate``` initially failed to solve the problem (I tried that first because of [this post](https://github.com/rstudio/keras/issues/869)). However, after posting I decided a good solution would be good to update all of my R packages to see if that did the trick. After using update.packages() to do that, I still had some weird missing package issues and had to individually update or reinstall a few packages. I believe ```reticulate``` was one of them, as well as a couple of random Tidyverse packages. Afterwards, the Keras installation function started working.
